kubernetes/zenko/charts/backbeat/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "8.0.18"
+appVersion: "8.0.19"
 description: A Helm chart for Zenko Backbeat
 name: backbeat
-version: 1.0.0
+version: 1.0.2
```

Comment:
we're skipping 1.0.1?

Reply:
It's open to debate. Me and @giacomoguiulfo were discussing this and couldn't decide whether keeping the Chart versions as their own version or mirroring the Zenko release version. No changes had been made to the backbeat chart which is why it hasn't changed until now.

Any thoughts? Would it be more confusing for the chart versions to have their own diverging versions or follow the Zenko release version?

Reply:
From my pov, having multiple diverging versions would be harder to follow, especially since our documentation doesn't get edited to keep up sometimes.